tests/config/pytest_modules.yml

```diff
@@ -2608,6 +2608,14 @@ unicycler:
   - modules/nf-core/unicycler/**
   - tests/modules/nf-core/unicycler/**
 
+universc:
+  - modules/nf-core/universc/mkref/**
+  - modules/nf-core/universc/mkgtf/**
+  - modules/nf-core/universc/**
+  - tests/modules/nf-core/universc/mkref/**
+  - tests/modules/nf-core/universc/mkgtf/**
+  - tests/modules/nf-core/universc/**
+
```

Comment:
Can we split these up at all, or do they all need to be tested if one change happens?

Reply:
These are tested jointly as is done for the "cellranger" module with similar functions.

```
cellranger/count:
  - modules/nf-core/cellranger/count/**
  - tests/modules/nf-core/cellranger/count/**
  - modules/nf-core/cellranger/mkref/**
  - tests/modules/nf-core/cellranger/mkref/**
  - modules/nf-core/cellranger/gtf/**
  - tests/modules/nf-core/cellranger/gtf/**

cellranger/gtf:
  - modules/nf-core/cellranger/gtf/**
  - tests/modules/nf-core/cellranger/gtf/**

cellranger/mkfastq:
  - modules/nf-core/cellranger/mkfastq/**
  - tests/modules/nf-core/cellranger/mkfastq/**

cellranger/mkgtf:
  - modules/nf-core/cellranger/mkgtf/**
  - tests/modules/nf-core/cellranger/mkgtf/**

cellranger/mkref:
  - modules/nf-core/cellranger/mkref/**
  - tests/modules/nf-core/cellranger/mkref/**
  - modules/nf-core/cellranger/gtf/**
  - tests/modules/nf-core/cellranger/gtf/**
```